client/templates/missingDamaged/missingDamaged.js

```diff
@@ -1,9 +1,13 @@
 Template.missingDamaged.onCreated(function () {
-  this.subscribe('ordersWithMissing/DamagedItems');
+  this.autorun(() => {
+    ReactionRouter.watchPathChange();
+    this.subscribe('ordersWithMissing/DamagedItems');
```

Comment:
The `/` here is a little confusing to me. Definitely works, but because of our previous convention, makes me think that it's coming from a package `ordersWithMissing` and that the subscription is `DamagedItems`